packages/cadl-ranch-specs/http/versioning/removed/main.tsp

```diff
@@ -66,6 +73,14 @@ model ModelV2 {
   unionProp: UnionV2;
 }
 
+model ModelV3 {
```

Comment:
Shouldn't `type` be added in `v1`, since we expect `v1` and `v2` to send a value for `type`, but not `v3`? If not, I don't really see how this test is different from `removed_v2`, since they're both testing if it's included for one version and removed in the next

Reply:
If we didn't specifically handle it with `@removed(Versions.V1)`, isn't the type property present by default in version V1? 
 This operation tests the combination of `@removed` and `@added`. According to @archerzz 's suggestion, this type property is added in the first version, removed in the second version, and added again in the third version, which still differs from V2.

Reply:
You've explicitly added `@added(v2)`, which makes it not present in `v1`

Reply:
> is it kind of a test of ABA problem?

I think so. If using @added() causes a property to not exist in another version, I'm not sure how to design a reasonable test case for testing this.

Reply:
Removing the `@added` should fix the issue, since by default everything is added in the first api version, unless you explicitly specify that it was added in a later version

Reply:
Removed.